packages/pnpm-sync-lib/src/pnpmSyncPrepare.ts

```diff
@@ -29,6 +30,11 @@ export interface IPnpmSyncPrepareOptions {
    */
   dotPnpmFolder: string;
 
+  /**
+   * An identifier that can be used to recognize the `pnpm-lock.yaml`
+   */
+  identifier?: string;
+
```

Comment:
I think it would also be useful to provide a way to delete unknown identifiers, for example if a Rush subspace was deleted.

The setting could be like:

```js
/** 
  * The identifier to be prepared by this operation.  If it an entry already exists with this name,
  * then it will be replaced.
  */
"identifier": "subspace2",

/** 
 * Instruct the API to delete any identifiers that don't appear in this list.
 */
"pruneIdentifiersExcept": [ "subspace1", "subspace2", "subspace3" ]
```

Since my idea here is basically a separate feature, maybe it could be introduced later as a separate PR.  Along with a corresponding CLI like:
```bash
pnpm-sync prepare --prune-identifiers-except subspace1,subspace2,subspace3
``` 
or 
```bash
pnpm-sync prune --except subspace1,subspace2,subspace3
```

Reply:
We need to pass `pnpm-lock.yaml` to `pnpm-sync prepare`. So in the below example won't work, unless we pass ALL `pnpm-lock.yaml` files to the command, which is not intended. 
```
pnpm-sync prepare --prune-identifiers-except subspace1,subspace2,subspace3
```

In the situation that Rush subspace was deleted, the identifiers related to this subspace will be deleted as well along with the regular `rush install/update` events